btcjson/chainsvrcmds.go

```diff
@@ -333,6 +333,19 @@ func NewGetInfoCmd() *GetInfoCmd {
 	return &GetInfoCmd{}
 }
 
+// GetMempoolEntryCmd defines the getmempoolentry JSON-RPC command.
+type GetMempoolEntryCmd struct {
+	TxID string
```

Comment:
Yep.. I added it to that on accident.  We can remove it but it doesn't really matter, it just gets ignored.